.github/workflows/ci.yml

```diff
@@ -25,29 +25,27 @@ on:
       - 'docs/**'
   workflow_dispatch:
 
-
 # Cancel redundant CI tests automatically
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-  build:
-    name: Run Tests
-    runs-on: ubuntu-latest
+  # reduced to one version that runs the codecov upload since it fails alot
```

Comment:
Does it really fail a lot? Or did it only fail yesterday when the repository got disabled in Codecov after going public? Until I activated it again.